packages/babel-plugin-transform-private-methods/test/fixtures/private-static-method-privateFieldsAsProperties/exfiltrated/output.js

```diff
@@ -3,7 +3,7 @@ var _privateStaticMethod = /*#__PURE__*/babelHelpers.classPrivateFieldLooseKey("
 class Cl {
   constructor() {
     if (exfiltrated === undefined) {
-      exfiltrated = babelHelpers.classPrivateFieldLooseBase(Cl, _privateStaticMethod)[_privateStaticMethod];
+      exfiltrated = babelHelpers.classPrivateFieldGetLoose(Cl, _privateStaticMethod);
```

Comment:
For methods, when the receiver is the class name, we can skip the helper call (similarly to how we do for the non-loose transform)

Reply:
![image](https://github.com/babel/babel/assets/30521560/741a638f-1fc7-4d94-958d-c3ba0832748b)
Unfortunately this will cause the test to fail. :)